l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -664,27 +656,25 @@ def _addGlobalDiscount(self, invoice_id, DatiGeneraliDocumento):
                         discount -= float(DiscRise.Importo)
                     elif DiscRise.Tipo == 'MG':
                         discount += float(DiscRise.Importo)
-            journal = self.get_purchase_journal(invoice.company_id)
-            credit_account_id = journal.default_credit_account_id.id
-            if not credit_account_id:
-                credit_account_id = self.env['ir.property'].with_context(
-                    force_company=invoice.company_id.id
-                ).get('property_account_expense_categ_id', 'product.category').id
+
+            company = self.env.user.company_id
```

Comment:
Che sia preferibile usare la `company_id` della fattura?

Reply:
Grazie della review!
Penso che in generale debbano coincidere, ma visto che si parla di righe da aggiungere alla fattura potrebbe avere più senso recuperarla dalla fattura stessa, modificato.